fitbenchmarking/fitting/controllers/dfogn_controller.py

```diff
@@ -0,0 +1,66 @@
+"""
+Implements a controller for DFO-GN
+http://people.maths.ox.ac.uk/robertsl/dfogn/
+"""
+
+import dfogn
+import numpy as np
+
+from fitbenchmarking.fitting.controllers.base_controller import Controller
+
+class DFOGNController(Controller):
+    """
+    Controller for the DFO-GN fitting software.
+    """
+
+    def __init__(self, problem, use_errors):
+        """
+        Initialises variable used for temporary storage.
+        """
+        super(DFOGNController, self).__init__(problem, use_errors)
+
+        self._soln = None
+        self._popt = None
+        self._pinit = None
+        self._use_errors = use_errors
```

Comment:
`use_errors` is stored in `self.use_errors` already as part of the base init

Reply:
Thanks -- fixed